tests/functional/stax/test_ui.py

```diff
@@ -0,0 +1,90 @@
+import pytest
+from ragger.navigator import NavIns
+from typing import Iterable, Union
+
+from .navigator import NavInsID
+
+
+def format_instructions(instructions: Iterable[Union[NavIns, NavInsID]]) -> Iterable[NavIns]:
+    return [NavIns(instruction) if isinstance(instruction, NavInsID) else instruction
+            for instruction in instructions]
```

Comment:
not needed anymore I guess.

Reply:
Yes it is. It's a direct result of the (re-)definition of the custom `NavInsID`. The `NavInsID` I use are _not_ the `ragger.NavInsID`, even if they _behave_ exactly the same.
So when Ragger checks `is this a NavIns or a NavInsID?` with `isinstance` on **my** `NavInsID`, the answer is **neither**, and it fails. So I have to convert my `NavInsID` into a ragger `NavIns`.

Reply:
I'll rename this `NavInsID` class just to avoid any confusion on this matter.